css-stlye/style.css

```diff
@@ -33,4 +34,65 @@ body {
     font-size: 12px;
     color: #b3afaf;
   }
-  /* End Header */
\ No newline at end of file
+  /* End Header */
+  /* Start Section*/
+  .gallery {
+    margin-top: 25px;
+    width: 100%;
+    height: 100%;
+  }
+  .gallery .img {
+    width: 100%;
+    height: 100%;
+    box-shadow: 1px 5px 6px #888888;
+    border-radius: 15px;
+    margin-bottom: 20px;
+  }
+  .gallery .img-gallery {
+    display: flex;
+  }
+  .gallery .img-gallery img {
+    border-radius: 15px;
+    box-shadow: 1px 5px 6px #888888;
+
+  }
+
+  .gallery .img-left :first-child {
+    margin-bottom: 23px;
+
+  }
+  .gallery .img-left img {
+    width: 162px;
+    height: 162px;
+  }
+  .gallery .img-right img {
+    width:120px ;
+    height: 352px;
+  }
+  .gallery .img-right :first-child {
+    width: 180px;
+  }
+
+   /*End Section */
+   /*Start Footer */
+  footer {
+    background-color: #fff;
+    text-align: center;
+    border-radius: 30px;
+    height: 92px;
+    width: 357px;
+    position: fixed;
+    bottom: 30%;
```

Comment:
The footer is floating on the page, use : 

```
position: fixed;
   left: 0;
   bottom: 0;
   width: 100%;
```

![Screenshot from 2021-06-26 10-57-55](https://user-images.githubusercontent.com/47992412/123506563-735a2900-d66d-11eb-9bd2-e57e4363b9ab.png)